AIToolbox/experiment_save/local_results_save.py

```diff
@@ -238,7 +238,7 @@ def save_experiment_results_separate_files(self, result_package, project_name, e
                                [train_hist_file_name, train_hist_file_local_path]]
 
         if save_true_pred_labels:
-            experiment_true_pred_labels_dict = {'y_true': result_package.y_true.tolist(), 'y_predicted': result_package.y_predicted.tolist()}
+            experiment_true_pred_labels_dict = {'y_true': result_package.y_true, 'y_predicted': result_package.y_predicted}
```

Comment:
removed .tolist()... be on guard if this brakes something in the result saving pipeline down the line. Based on the tests I executed everything still works fine.

To list was removed to enable saving both lists as well as dicts in case you used MultipleResultPackageWrapper